scripts/setup/config.ps1

```diff
@@ -45,13 +45,38 @@ Param (
     
     [parameter()]
     [string]
-    $Destination
+    $Destination,
+
+    [parameter()]
+    [string]
+    $IISAdminOwnersGroup = "IIS Administration API Owners",
+
+    [parameter()]
+    [switch]
+    $LegacyConfigurations
 )
 
 # Name of file we place installation data in
 $INSTALL_FILE = "setup.config"
 $IISAdminSiteName = "IISAdmin"
 
+## Ensure the local gorup exists
+function EnsureGroup($group) {
+    if (!(Get-LocalGroup -Name $group -ErrorAction SilentlyContinue)) {
+        New-LocalGroup -Name $group | Out-Null
+    }
+}
+
+## ensure the member/group exists in the specified group
+function EnsureMember($group, $userOrGroup) {
```

Comment:
`EnsureLocalGroupMember`  would probably be better to follow the powershell commandlets.

Reply:
updated